integration-cli/requirements.go

```diff
@@ -25,8 +30,37 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	NativeExecDriver = TestRequirement{
+		func() bool {
+			return strings.HasPrefix(daemonExecDriver, "native")
+		},
+		"Test requires the native (libcontainer) exec driver.",
+	}
 )
 
+func init() {
+	// get daemon info
+	body, err := sockRequest("GET", "/info", nil)
```

Comment:
can we not do this lazy fetch inside the TestCondition func and cache there? when I run a single test case with `TESTFLAGS='-test.run XXX'` on my machine, I probably don't want to slow down of test bootstrap any longer.